src/exec/util/unit.go

```diff
@@ -27,26 +27,26 @@ const (
 	DefaultPresetPermissions os.FileMode = 0644
```

Comment:
Looks like this one could be removed now you're defining and using DefaultFilePermissions